### ISSUE TITLE
Add logging to `gs://` filepaths without requiring a `tensorflow` installation.

### ISSUE THREAD

**Issue by LarsDu** [tensorflow/tensorboard#6298]
I support a team that is primarily using PyTorch and Google Cloud within a kubernetes cluster.
We discovered that `tensorboard` can log to S3:// (that's Amazon AWS!) without requiring a full `tensorflow` installation, but GCP users must either use `tensorboardX` to log to GCS, or do a full `tensorflow` installation alongside `tensorboard`. This brings in a lot of problematic dependencies and can make it impossible to incorporate a tensorboard install in a large project managed by a dependency resolver such as poetry or conda.

The tensorboardX project current support logging to GCS paths, and tensorboard currently supports logging to s3:// paths. 

As a Google created open sourced project, I only feel it should be natural to support tensorboard logging to GCS (gs://) paths without requiring a tensorflow install.

**Comment by bileschi** [comment]
@yatbear is this something that the TensorFlow independence effort would help with, or would a separate effort be needed?

**Comment by yatbear** [comment]
Hi @LarsDu,

Thanks for the feedback! TensorBoard does support GCS paths (https://github.com/tensorflow/tensorboard/blob/master/tensorboard/util/io_util.py#L19). To better understand the your are encountering - are you blocked by the fact that GCP requires a full TensorFlow runtime dependency in order to use TensorBoard? And you want to be able the `notf` version TensorBoard on GCP?

**Comment by LarsDu** [comment]
On second thought this might be an issue with `torch` and the way its tensorboard plugins handle things. I'm closing this issue for now. What is the `notf` version of Tensorboard?

**Comment by yatbear** [comment]
Hi @LarsDu,

The `notf` version is just TensorBoard without the full TF installation.
